src/Components/Superiority.jsx

```diff
@@ -0,0 +1,92 @@
+import { Flex, Text } from "@sparrowengg/twigs-react";
+
+import AlertIcon from "./../asset/alert-circle.svg";
+import AlertIcon2 from "./../asset/alert-circle2.svg";
+import Frame from "./../asset/frame2.png";
+import Link from "./../asset/link.svg";
+
+import { stockData } from "../utils/data";
+
+import StockTrading from "./StockTrading";
+import { useSelector } from "react-redux";
+import { LIGHT_MODE } from "../redux/light-dark/lightDarkTypes";
+
+const Superiority = () => {
+  const mode = useSelector((store) => store.lightDARK_MODE.lightDARK_MODE);
+
+  return (
+    <Flex
+      flexDirection="column"
+      alignItems="flex-start"
+      css={{
+        padding: "34px",
+      }}
+    >
+      <Flex alignItems="center" gap="10px">
+        <Text
+          size="md"
+          weight={"bold"}
+          css={{
+            lineHeight: "28px",
+            color: "$textPrimary",
+          }}
+        >
+          Superiority
+        </Text>
+        <img src={mode == LIGHT_MODE ? AlertIcon : AlertIcon2} alt="Alert" />
+      </Flex>
+
+      <img
```

Comment:
Use styled method from twigs for `img` tag, to and make use of css prop for stylings.